internal/build/container_resolver.go

```diff
@@ -0,0 +1,60 @@
+package build
```

Comment:
this would be a great thing to NOT lump in the `build` package, imo -- maybe a separate `containers` package? or the `docker` package? but it's not reallllly the domain of `build` imo.